plans/logs.tf

```diff
@@ -0,0 +1,42 @@
+resource "azurerm_resource_group" "logs" {
+    name     = "${var.prefix}logs"
+    location = "${var.location}"
+    tags {
+        env = "${var.prefix}"
+    }
+}
+
+resource "azurerm_storage_account" "logs" {
+    name                = "${var.prefix}logs"
+    resource_group_name = "${azurerm_resource_group.logs.name}"
+    location            = "${var.location}"
+    account_type        = "Standard_GRS"
+    depends_on          = ["azurerm_resource_group.logs"]
+    tags {
+        env = "${var.prefix}"
+    }
+}
+
+resource "azurerm_storage_share" "logs" {
+    name = "logs"
+    resource_group_name     = "${azurerm_resource_group.logs.name}"
+    storage_account_name    = "${azurerm_storage_account.logs.name}"
+    quota                   = 50
+    depends_on              = ["azurerm_resource_group.logs","azurerm_storage_account.logs"]
+}
+
+resource "azurerm_template_deployment" "logs"{
+    name  = "${var.prefix}logs"
+    resource_group_name = "${ azurerm_resource_group.logs.name }"
+    depends_on          = ["azurerm_resource_group.logs"]
+    parameters = {
+        omsWorkspaceName = "${var.prefix}logs"
+        omsRegion = "${var.location}"
+        existingStorageAccountName = "${azurerm_storage_account.logs.name}"
+        existingStorageAccountResourceGroupName = "${azurerm_resource_group.logs.name}"
+        #table = ""
+    }
+    #deployment_mode = "Complete"
+    deployment_mode = "Incremental"
```

Comment:
If I understand correctly terraform documentation, we must use incremental
Otherwise we will destroy other resources. 

_deployment_mode - (Required) Specifies the mode that is used to deploy resources. This value could be either Incremental or Complete. Note that you will almost always want this to be set to Incremental otherwise the deployment will destroy all infrastructure not specified within the template, and Terraform will not be aware of this._